config/initializers/sidekiq.rb

```diff
@@ -7,9 +7,9 @@
 end
 
 Sidekiq.configure_server do |config|
-  config.redis = { url: sidekiq['url'], namespace: sidekiq['namespace'] }
```

Comment:
Do you think a migration is necessary for things still on the sidekiq namespace? e.g. running or scheduled jobs

Reply:
@stevenbarragan  Sidekiq 7.x requires to move away from namespace, please find the other details in [SRCH-4293](https://cm-jira.usa.gov/browse/SRCH-4293). Regarding the migration, existing sidekiq instance will be stopped and new instance will be pointed to 2 going forward, refer to [SRCH-4308 ](https://cm-jira.usa.gov/browse/SRCH-4308)